source/utils/types.ts

```diff
@@ -122,6 +122,7 @@ export interface AgentByProtocol {
 export interface Options extends Omit<https.RequestOptions, 'agent'> {
 	host: string;
 	body: string | Buffer | ReadableStream;
+	userData?: unknown;
```

Comment:
Nitpick, but I would place it at the bottom, above: `[key: string]: unknown;`.

Reply:
Sure.